### PR TITLE
feat: delete server side fields in kubernetes runtime

### LINKED ISSUE #705: Feat: ignore `managedFields` of K8s resources in kusion preview
### What would you like to be added?

Ignore `attributes.metadata.managedFields` of K8s resources in the kusion preview command

### Why is this needed?

image
 As shown in the image above, the presence of the managedFields attribute hinders users from focusing on the changes made by individuals.

### PR DESCRIPTION
**Pull request by liu-hm19**
#### What type of PR is this?

/kind feature
#### What this PR does / why we need it:
This PR deletes server-side fields in Kubernetes runtime for a more precise and clean resource object. 

- Before: 
image

- After: 
image

#### Which issue(s) this PR fixes:

Fixes #705 

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., design docs, usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/engine/runtime/kubernetes/kubernetes_runtime.go [hunk 1]
```diff
@@ -225,17 +233,14 @@ func (k *KubernetesRuntime) Import(ctx context.Context, request *runtime.ImportR
...
-					Resource: nil,
-					Status:   v1.NewErrorStatusWithCode(v1.IllegalManifest, err),
-				}
+		if err := normalizeService(ur); err != nil {
+			return &runtime.ImportResponse{
+				Resource: nil,
+				Status:   v1.NewErrorStatusWithCode(v1.IllegalManifest, err),
 			}
 		}
-		normalizeServerSideFields(ur)
```
Comment (SparkYuan) [inline pkg/engine/runtime/kubernetes/kubernetes_runtime.go:238]: Why did you delete this line? `normalizeServerSideFields` and `normalizeService` are not the same.

Comment (liu-hm19) [reply pkg/engine/runtime/kubernetes/kubernetes_runtime.go:238]: Currently, only the `Import` method handles `normalizeServerSideFields` for resource objects obtained from the cluster. This PR adds the process of `normalizeServerSideFields` in both `Read` and `Apply`, hence the `Import` method obtaining cluster resources via `Read` should no longer require additional processing.

**Review by SparkYuan — APPROVED** [review]
lgtm
